pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/InTransformFunction.java

```diff
@@ -56,6 +56,8 @@ public String getName() {
 
   @Override
   public void init(List<TransformFunction> arguments, Map<String, ColumnContext> columnContextMap) {
+    // TODO: Proper support for null in groovy functions.
```

Comment:
Can you elaborate this TODO a bit more. What's missing to provide proper support?